students/erica_edwards/Lesson04/mailroom2.py

```diff
@@ -0,0 +1,113 @@
+
+from operator import itemgetter
+
+donors = {'Sandy Pie' : [75],
+          'Judy Smith': [75, 100, 1000],
+          'Mike Jones': [75, 1000],
+          'Joe Smith' : [75, 100, 2000],
+          'Kelly Blue': [75, 150, 275]}
+
+
+def menu():
+    while True:
+        print()
+        print('''Menu:
+                    1. Send a Thank You.
+
+                    2. Send all Thank You's
+
+                    3. Create a Report.
+
+                    4. Quit.''')
+        print()
+        response = input("Please enter a number to make your selection. ")
+
+        switch_func_dict = {0: ask_donor_name, 1: multi_thanks, 2: totals, }
+        if response == '1':
+            switch_func_dict.get(0)()
+        if response == '2':
+            switch_func_dict.get(1)()
+        if response == '3':
+            switch_func_dict.get(2)()
+        if response == '4':
+            break
```

Comment:
the point of switch dict is to get rid of the if statements (besides these should be if/elif)

your code should look like this:
```
while True:
    response = input('')
    switch_func[response]()
```

Reply:
you should be your quit option to a quit function that calls sys.exit